src/db/schema/qbInvoiceSync.ts

```diff
@@ -1,16 +1,30 @@
+import { InvoiceStatus } from '@/app/api/core/types/invoice'
 import { timestamps } from '@/db/helper/column.helper'
 import { pgTable as table } from 'drizzle-orm/pg-core'
 import * as t from 'drizzle-orm/pg-core'
 import { createInsertSchema, createSelectSchema } from 'drizzle-zod'
 import { z } from 'zod'
 
+export function enumToPgEnum<T extends Record<string, any>>(
+  myEnum: T,
+): [T[keyof T], ...T[keyof T][]] {
+  return Object.values(myEnum).map((value: any) => `${value}`) as any
```

Comment:
Using `as any` here defeats the purpose of typing the return as a tuple... if you're going to do type casting, better to do
```suggestion
  return Object.values(myEnum).map((value: any) => `${value}`) as  [T[keyof T], ...T[keyof T][]];
```